local-compose.yml

```diff
@@ -6,7 +6,7 @@ services:
     build:
       context: ./web/
       dockerfile: Dockerfile-tests
-    command: "python manage.py runserver 0.0.0.0:7000"
+    command: "python3 manage.py runserver 0.0.0.0:7000"
```

Comment:
Here and below:
Please clarify where do we need trailing `3` in the words `python` and `pip` and where it could be omitted.